### ISSUE TITLE
Jdk 17 - TestNG/Guice dependency version

### ISSUE THREAD

**Issue by MBogdanski** [prashant-ramcharan/courgette-jvm#324]
Hello,

When I want to use jdk 17, I have a problem with dependency injection using Guice. I want to try update Guice version to 5.1.0, but in Courgette-JVM, TestNG in 7.1.0 version is used, which is using Guice in version 4.1.0. 
So Gradle want to use 5.1.0_no_aop version to fullfil dependency on 4.1.0 Guice version, but it's failing since Guice from version 5.0+ is not divided to aop/no_aop version.
You think that can may be the problem? Maybe updating the dependency TestNG to 7.5(which use Guice in 5.1.) will resolve the issue?

**Comment by prashant-ramcharan** [comment]
Hello

You should be able to use another version of TestNG by excluding it from the Courgette dep tree and providing your own version within your project.

```gradle
    testCompile('io.github.prashant-ramcharan:courgette-jvm:6.4.1') {
        exclude group: 'org.testng'
    }

    testCompile "org.testng:testng:7.5" 
````

**Comment by MBogdanski** [comment]
Yes, does similar, Just excluded guice dependency. Thanks for info

**Comment by MBogdanski** [comment]
Resolved by excluding dependency
